crates/bin/src/main.rs

```diff
@@ -263,7 +105,7 @@ async fn entry(jobserver_client: LazyJobserverClient) -> Result<()> {
     };
 
     // Load options
-    let mut opts = Options::parse_from(args);
+    let mut opts = args::Args::parse_from(args);
     if opts.quiet {
         opts.log_level = LevelFilter::Off;
     }
```

Comment:
Can we also extract line 88-130 into the new mod `args`, as an associated function `Args::parse`?
Then we also don't need to import `clap::Parser`.